### DISCUSSION TITLE
[FEATURE]: Add ability to update marketing information

### DISCUSSION THREAD

**Discussion by johnbie** [etsy/open-api#730] [Ideas]
### Is there an existing ticket for this?

- [X] I have searched the existing tickets

### Is your feature request related to a problem? Please describe

There are no marketing endpoints for open-api, and my clients are quite annoyed that they have to manually check and update their advertisement information. I'm sure that the ability to read and write this information will not only increase their revenue, but also ETSY's.

### Describe the solution you'd like

Here are some things I want:
 - I want to be able to use the api to check and increase my advertising budget when it runs out SO THAT my shop is always being advertised. 
 - I want to be able to use the api to see how much I spent on advertising SO THAT I can adjust my marketing budget.
 - I want to be able to see which order items were bought through advertisement SO THAT I can adjust the advertised listings.
 - I want to be able to get the list of advertised listings and change its advertising status SO THAT I am always advertising the most well-liked items.

### Describe any alternatives you've considered

The alternative I've considered is directly crawling the said information with a crawler. I have not gone this path as I suspected that ETSY would not like such side-door implementation and I don't want to risk banning my client's accounts.

However, it would be quite easy to make such requests appear organic, provided that it happens at a random and normal-looking interval and with a browser-based user agent.

**Comment by etsyachristensen** [comment]
@johnbie I'm moving this ticket to an Idea Discussion so we can continue to track it after the issues board is closed.

**Comment by sailai** [comment]
It is very import for developer to enrich their seller application.

**Comment by supermandee** [comment]
any update on this? or how do i access this info using open API v3?
